pkgs/development/tools/osh-tool/default.nix

```diff
@@ -29,6 +29,5 @@ nimPackages.buildNimPackage rec {
     license = lib.licenses.agpl3;
     platforms = nim.meta.platforms;
     maintainers = with maintainers; [ sourceindex ];
-
```

Comment:
You can squash such minor typo corrections, as if they never existed.

Reply:
Thanks. Indeed seems right.